### DISCUSSION TITLE
GitHub Desktop on Linux: can't select Pycharm as default editor

### DISCUSSION THREAD

**Discussion by AB-Stromer** [community/community#46325] [Other Feature Feedback, Questions, & Ideas]
Hi, I'm new to GitHub, have established my first repository for my Python + Qt code, using Pycharm as IDE, have installed GitHub Desktop on Manjaro Linux. Works well except:
In "Options" I cannot select "Pycharm" IDE for "Default Editor"! 

Yes, I am aware that Linux is not a supported platform for GitHub, but anyway, allow me to just ask in case anyone has experience and might help with advice

Thanks, best regards!

**Comment by chris-c-thomas** [comment]
Are you trying the .AppImage version from here? - https://github.com/shiftkey/desktop

I'd imagine you're unlikely to get a solution here on GitHub discussions. If you are using /u/shiftkey (he does work for them) I'd try posting in the [issues](https://github.com/shiftkey/desktop/issues) section of that repo. Seems there is some other weird behavior at times like VS Code opening instead of the file manager: https://github.com/shiftkey/desktop/issues/804

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @AB-Stromer ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
